trieste/models/gpflow/sampler.py

```diff
@@ -91,11 +116,25 @@ def sample(self, at: TensorType, *, jitter: float = DEFAULTS.JITTER) -> TensorTy
         mean, var = self._model.predict(at[..., None, :, :])  # [..., 1, 1, L], [..., 1, 1, L]
         var = var + jitter
 
-        if not self._initialized:
-            self._eps.assign(
-                tf.random.normal([self._sample_size, tf.shape(mean)[-1]], dtype=tf.float64)
-            )  # [S, L]
+        def sample_eps() -> tf.Tensor:
             self._initialized.assign(True)
+            if self._qmc:
+                skip = IndependentReparametrizationSampler.skip
+                IndependentReparametrizationSampler.skip.assign(skip + self._sample_size)
+                normal_samples = qmc_normal_samples(
+                    tf.TensorShape(self._sample_size), mean.shape[-1], skip
```

Comment:
Now I understand what this is doing, it creates a space-filling design in the space the dimension of the number of latents. In most cases this will just be a 1d sequence (single latent), in other cases it might be mildly beneficial to have this space-filling structure in the latent space, i.e. with 2 latents you would maximise the diversity of the coupled values of `f1` and `f2`.